### PR TITLE
[ashell] Fix to allow command line ashell to be built.

### PR DESCRIPTION
**Pull request by brianjjones**
Fix for #1257

Signed-off-by: Brian J Jones <brian.j.jones@intel.com>

### CODE REVIEW

**Comment by grgustaf** [comment]
Rather than duplicate most of the JSON in these two files, let's leave a zjs_ashell.json with all the core settings except for the one that changes. Then just make the _cli and _ide files only contain "depends" on "ashell" and specify the one setting that is different.

File: src/zjs_ashell_ide.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "module": "ashell_ide",
+    "depends": ["aio", "arduino101_pins", "ble",
+                "common", "console", "fs", "gpio", "grove_lcd", "i2c",
+                "performance", "pwm", "sensor_accel", "sensor_gyro",
+                "sensor_light", "sensor_temp", "spi"],
```
Comment (grgustaf) [inline src/zjs_ashell_ide.json]: Based on my previous comment, in the _cli and _ide versions I think this can just contain "ashell" and nothing else.

**Comment by brianjjones** [comment]
Moved all the common code to zjs_ashell.json

**Comment by grgustaf** [comment]
+1 looks great, thanks...

**Pull request merged**